modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/itemselector/plugin.js

```diff
@@ -250,13 +250,13 @@
 						callback(imageSrc, selectedItem);
 					}
 					else {
-						var imageElement = new CKEDITOR.dom.element.createFromHtml(
-							'<img src="' + imageSrc + '">'
-						);
+						if (IE9AndLater) {
```

Comment:
I don't know why we even have any `IE9AndLater` check in here at all... Shouldn't we be removing this branch from `master`? I would expect to see this only on 7.3.x and prior (provided we backport it there).

Reply:
Yes, good point @wincent  it's there because we never removed it.
But we should

Reply:
I don't mind removing it for master, and keep it only for 7.3.x 
Regards.

Reply:
I pushed a version without IE9AndLater that seems to work for IE and Edge as well,
Regards